src/main/java/fr/pickaria/pterodactylpoweraction/StartingServer.java

```diff
@@ -28,13 +29,17 @@ public class StartingServer implements ForwardingAudience {
     private final Logger logger;
     private final Messager messager;
     private final AtomicBoolean isStarting = new AtomicBoolean(false);
+    private ServerStartBossBar bossBar;
+    private static final Set<StartingServer> INSTANCES = ConcurrentHashMap.newKeySet();
```

Comment:
Storing static instances is usually a bad practice as it does not comply with the SOLID principles.